brave/brave-core
# Adds claim api for vg

Resolves https://github.com/brave/brave-browser/issues/8354

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
Plan 1:
- enable rewards
- claim ad grant and ugp
- connect to KYC uphold account
- make sure that ad grant was transferred  to uphold while ugp stayed as VG


Plan 1:
- enable rewards
- connect to KYC uphold account
- disconnect uphold
- claim ad grant and ugp
- make sure that ad grant was transferred  to uphold while ugp stayed as VG

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
